scalikejdbc-streams/src/main/scala/scalikejdbc/streams/DatabasePublisherSettings.scala

```diff
@@ -5,20 +5,38 @@ import scalikejdbc._
 /**
  * Settings for DatabasePublisher.
  */
-class DatabasePublisherSettings[A](
-  val connectionPoolName: Any,
+private[streams] class DatabasePublisherSettings[A](
+  /**
+   * Connection pool name.
+   */
+  val dbName: Any,
```

Comment:
I suggested to use connectionPoolName but in the point of the view of consistency with existing code, I rethought dbName is also fine. Sorry for bothering you.